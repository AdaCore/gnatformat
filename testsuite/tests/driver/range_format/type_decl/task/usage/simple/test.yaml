description: ObjectDecl formatting for line selection.
driver: gnatformat
args:
  [
    "test.ads",
    "--range-format",
    "-SL 141",
    "-EL 142",
    "--pipe",
  ]
