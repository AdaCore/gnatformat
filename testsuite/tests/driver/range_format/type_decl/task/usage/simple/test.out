test.ads(<ObjectDecl ["Agent"] test.ads:141:4-141:22>) - 141:1-141:22
^
   Agent : Server;$

